services/web/server/src/simcore_service_webserver/conversations/_socketio.py

```diff
@@ -106,7 +106,7 @@ async def _send_message_to_recipients(
         ...
 
 
-async def notify_conversation_created(
+async def notify_via_socket_conversation_created(
```

Comment:
MINOR: `socketio`. AFAIK it is websockets or socketio.  

* **Socket:** a low-level TCP/UDP endpoint for sending and receiving raw bytes.
* **WebSocket:** a full-duplex messaging protocol that operates over a TCP socket.
* **Socket.IO:** a higher-level event-based framework built on WebSockets with fallback transports and added features.